src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/RssLink.java

```diff
@@ -2,14 +2,15 @@
 
 import com.google.common.base.Preconditions;
 import lombok.NonNull;
+import org.apache.commons.lang3.StringUtils;
 import pl.tomaszdziurko.jvm_bloggers.blogs.domain.Blog;
 
 public class RssLink {
 
     private final Blog blog;
 
     public RssLink(@NonNull Blog blog) {
-        Preconditions.checkArgument(blog.getRss() != null, "Rss link can not be null");
+        Preconditions.checkArgument(StringUtils.isNotBlank(blog.getRss()), "Rss link can not be null");
```

Comment:
You have updated precondition but the message is still outdated and misleading. It should be sth. like  `Rss link can not be null nor empty`.